portfolio/src/main/webapp/script.js

```diff
@@ -15,58 +15,68 @@
-  e.preventDefault();
-  const response = await fetch('/comment', {
-    method: 'POST',
-    body: new URLSearchParams(new FormData(e.target)),
+document
+  .querySelector(".comment-form")
+  .addEventListener("submit", async function (e) {
+    e.preventDefault();
+    const response = await fetch("/comment", {
+      method: "POST",
+      body: new URLSearchParams(new FormData(e.target)),
+    });
+    const json = await response.json();
+    if (json.status == "error") {
+      console.log("Could not submit comment");
+      return;
+    }
+    appendComment(document.querySelector(".comment-box").value);
   });
-  const json = await response.json();
-  if (json.status == "error") {
-    console.log('Could not submit comment');
-    return;
-  }
-  appendComment(document.querySelector('.comment-box').value);
-});
 
 async function getComments(num) {
   const response = await fetch(`/comment?num=${num}`);
   const json = await response.json();
-  const commentContainer = document.querySelector('.comments');
-  commentContainer.innerHTML = '';
+  const commentContainer = document.querySelector(".comments");
+  commentContainer.innerHTML = "";
   for (const comment of json.comments) {
     appendComment(comment);
   }
 }
 
 function rerequestComments() {
-  let numCommentsElem = document.querySelector('.num-comments');
+  let numCommentsElem = document.querySelector(".num-comments");
   let num = numCommentsElem.options[numCommentsElem.selectedIndex].value;
   getComments(num);
 }
 
 function toggleComments() {
-  let comments = document.querySelector('.comments');
-  comments.style.display = (comments.style.display == 'none') ? 'block' : 'none';
+  let comments = document.querySelector(".comments");
+  comments.style.display = comments.style.display == "none" ? "block" : "none";
 }
 
 function appendComment(value) {
-  const commentContainer = document.querySelector('.comments');
-  let liComment = document.createElement('li');
+  const commentContainer = document.querySelector(".comments");
+  let liComment = document.createElement("li");
   liComment.innerText = value;
-  liComment.className = 'comment';
+  liComment.className = "comment";
   commentContainer.append(liComment);
 }
 
 async function deleteComments() {
-  const response = await fetch('/delete', {
-    method: 'POST',
+  const response = await fetch("/delete", {
+    method: "POST",
   });
   const json = await response.json();
   if (json.status == "error") {
-    console.log('Could not delete comments');
+    console.log("Could not delete comments");
     return;
   }
-  const commentContainer = document.querySelector('.comments');
-  commentContainer.innerHTML = '';
-};
+  const commentContainer = document.querySelector(".comments");
+  commentContainer.innerHTML = "";
+}
+
+/** Creates a map showing Columbia University and adds it to the page. */
+function createMap() {
+  const map = new google.maps.Map(document.getElementById("map"), {
```

Comment:
no need to store a reference `const map =` -- since you don't use the value anywhere.